docs/pages/installation/releases.md

```diff
@@ -13,6 +13,13 @@ A list of the [releases of Meshery](https://github.com/layer5io/meshery/releases
     <th style="white-space: nowrap;">Release Date</th>
   </thead>
   <tbody>
+      <tr>
+      <td class="centered">0.3.10</td>
+      <td>
+        <em>Mesheryctl</em> - Introduces mesheryctl perf command. See Meshery CLI Commands & Documentation for reference.
+      </td>
+      <td>Mar 9, 2020</td>
+    </tr>
```

Comment:
This looks good here.

On around [line 87 or so](https://github.com/layer5io/meshery/pull/721/files#diff-a040c9f47c27f91312f077ce372f9b92R133), it would be nice to add a row to separate the v0.3.x releases from the v0.2.x releases.